src/FACT_internal.c

```diff
@@ -1182,7 +1183,15 @@ void FACT_INTERNAL_ActivateEvent(
 			/* Add/Replace */
 			if (evt->value.equation.flags & 0x01)
 			{
-				evtInst->value += svResult;
+				if(	evt->type == FACTEVENT_PITCH ||
+					evt->type == FACTEVENT_PITCHREPEATING	)
+				{
+					evtInst->value = trackInst->evtPitch + svResult;
+				}
+				else
+				{
+					evtInst->value = trackInst->evtVolume + svResult;
+				}
 			}
```

Comment:
I have a test case which has a pitch and playwave event at timestamp 0. and then a second later another pitch event that does an add operation.  In this case the first set pitch value is lost as it's stored with the event.  So for additive cases we need to get the current pitch from the track or we'll just start back at 0.